<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qwad</class>
 <widget class="QMainWindow" name="Qwad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qwad - Empaquetador y desempaquetador de WADs</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Qwad.qrc">
    <normaloff>:/icons/wad.png</normaloff>:/icons/wad.png</iconset>
  </property>
  <property name="locale">
   <locale language="Spanish" country="Spain"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Desempaquetar</string>
     </attribute>
     <widget class="QLineEdit" name="MuestraRutaWad">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>241</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="BotonRutaWad">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Ruta hacia el archivo WAD que quieres desempaquetar</string>
      </property>
      <property name="text">
       <string>Archivo WAD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="MuestraRutaExtraer">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>241</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="BotonRutaExtraer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Carpeta donde se almacenaran los contenidos del archivo WAD</string>
      </property>
      <property name="text">
       <string>Destino</string>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="Desempaqueta">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Desempaquetar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Empaquetar</string>
     </attribute>
     <widget class="QCheckBox" name="updateTMD">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>SÃ³lo necesario en caso de que hayas modificado el WAD original</string>
      </property>
      <property name="text">
       <string extracomment="Necesario si has modificado los contenido dels wad original">Actualizar TMD</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="MuestraRutaEmpaquetado">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>241</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="MuestraRutaDesempaquetado">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>241</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="BotonRutaEmpaquetado">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Carpeta y nombre del nuevo WAD</string>
      </property>
      <property name="text">
       <string>Destino</string>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="BotonRutaDesempaquetado">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Ruta hacia la carpeta que empaquetar en WAD</string>
      </property>
      <property name="text">
       <string>Carpeta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Empaqueta">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Empaquetar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de_Qwad"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_Wii_Signer"/>
   </widget>
   <addaction name="menuAyuda"/>
  </widget>
  <action name="actionAcerca_de_Qwad">
   <property name="icon">
    <iconset resource="../Qwad.qrc">
     <normaloff>:/icons/wad.png</normaloff>:/icons/wad.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de Qwad</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="../Qwad.qrc">
     <normaloff>:/icons/qt4.png</normaloff>:/icons/qt4.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_Wii_Signer">
   <property name="text">
    <string>About Wii Signer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Qwad.qrc"/>
 </resources>
 <connections/>
</ui>
